README.md

```diff
@@ -130,12 +132,29 @@ https://docs.github.com/en/rest/licenses.
 **Inline example**: `deny-licenses: LGPL-2.0, BSD-2-Clause`
 
 **YAML example**:
+
 ```yaml
 deny-licenses:
   - LGPL-2.0
   - BSD-2-Clause
 ```
 
+### allow-ghsas
+
+Add a custom list of GHSA ids you do not want the action to block on.
```

Comment:
```suggestion
Add a custom list of GitHub Advisory IDs that can be skipped during detection. 
```

Reply:
Tiny nit: the Action does not block anything, it just reports a failure. Enforced checks (repo admin feature I think) is what blocks the actual PR.